functions/cors-go/README.md

```diff
@@ -25,3 +35,21 @@ curl -i -X OPTIONS <function URL>
 # Get request
 curl -i -X GET <function URL>
 ```
+
+The result should be similar to:
+
+```console
+HTTP/1.1 200 OK
+access-control-allow-headers: *
+access-control-allow-methods: *
+access-control-allow-origin: *
+content-length: 44
+content-type: text/plain
+date: Tue, 17 Jan 2023 15:56:52 GMT
+x-envoy-upstream-service-time: 10
+server: envoy
+
+This function is allowing most CORS requests
+```
+
+You can also check the result of your function in a browser. It should be "This is checking CORS".
```

Comment:
`"This is checking CORS"` -> `"This function is allowing most CORS requests"` ? 🤔

Reply:
Solved.